dhall/src/Dhall.hs

```diff
@@ -1860,6 +1857,34 @@ instance Inject a => Inject (Data.HashSet.HashSet a) where
 
 instance (Inject a, Inject b) => Inject (a, b)
 
+{-| Inject a `Data.Map` to a @Prelude.Map.Type@
+
+>>> prettyExpr $ embed inject (Data.Map.fromList [(1 :: Integer, True )])
```

Comment:
Usually for examples I try to use `Natural` instead of `Integer` (to encourage more people to use `Natural`)